### PR TITLE
style(colors): updates the colors to sage 3 colors

### PR DESCRIPTION
**Pull request by cmcfadzean**
re #490

## Description
Updates the colors to use the new sage 3 colors. 

## Testing in Sage

See Design > Colors and elsewhere colors are applied.

## Testing in `kajabi-products`

(LOW) Color palette adjusted to latest specs. Color changes will occur as a result in `kajabi-products` but no adverse effects expected.

### CODE REVIEW

File: packages/sage-assets/lib/stylesheets/tokens/_color_palette.scss [hunk 1]
```diff
@@ -10,58 +10,58 @@
 ///
 $sage-colors: (
   primary: (
-    100: #e6f4fe,
-    200: #8ecafb,
+    100: #E6F4FE,
```
Comment (philschanely) [inline packages/sage-assets/lib/stylesheets/tokens/_color_palette.scss:13]: @cmcfadzean can you please adjust all letters to lower-case in keeping with the precedent before? 🙏

**Review by philschanely — COMMENTED** [review]
Thanks @cmcfadzean one short comment for adjustment. 👍 otherwise.

File: packages/sage-assets/lib/stylesheets/tokens/_color_palette.scss [hunk 2]
```diff
@@ -82,12 +82,12 @@ $sage-colors: (
 );
 
 ///
-/// Sage breakpoints token utility
+/// sage breakpoints token utility
```
Comment (philschanely) [inline packages/sage-assets/lib/stylesheets/tokens/_color_palette.scss:85]: haha looks like a plugin challenged all letters to lowercase now 😂

**Review by philschanely — APPROVED** [review]
Not blocking, but it seems the uncaps you applied was a bit greedy :)

**Review by QuintonJason — APPROVED** [review]
nice!

File: packages/sage-assets/lib/stylesheets/tokens/_color_palette.scss (hunk 2 above)
Comment (cmcfadzean) [reply packages/sage-assets/lib/stylesheets/tokens/_color_palette.scss:85]: hahaha

**Pull request merged**